### DISCUSSION TITLE
React native examples

### DISCUSSION THREAD

**Discussion by marisancans** [react-hook-form/react-hook-form#3046] [Q&A] [answered]
Hi there, I have a need for forms in react native, but i'm having difficulties finding even a simple example code. I understand that 
 react-hook-form is working on native too, right? Can anyone supply a sample here, preferably in typescript, thanks!

**Comment by marcusflat** [comment, marked as answer]
Hi @marisancans , have you checked this section of [documentation](https://react-hook-form.com/get-started#ReactNative)? 
[Code Examples](https://github.com/react-hook-form/react-hook-form/tree/master/examples), [TS](https://react-hook-form.com/ts)

**Reply by marisancans** [reply]
Yes, I have, but these examples don't work on react-native mobile apps. There are tags like `<form>` and` <input> `that does not exist in react-native. I have to instead use react-native components, but figuring out what goes where in TS is hell.

**Reply by bluebill1049** [reply]
https://react-hook-form.com/get-started#ReactNative

**Reply by marisancans** [reply]
That seems to be the only example. Well, will have to suffice. Thanks

**Reply by yossimenash** [reply]
did you find any examples? 
i am looking for some too..
